src/prefect/tasks/airbyte/__init__.py

```diff
@@ -1,6 +1,6 @@
 """
-This module contains a task for triggering [Airbyte](https://airbyte.io/) connection sync jobs
+This module contains a task for triggering [Airbyte](https://airbyte.io/) connection sync jobs as well as a configuration export
 """
-from .airbyte import AirbyteConnectionTask
+from .airbyte import AirbyteConnectionTask, AirbyteConfigurationExport, AirbyteClient
```

Comment:
We don't need to expose `AirbyteClient` as part of the public API. It's just for internal use within our tasks.
```suggestion
from .airbyte import AirbyteConnectionTask, AirbyteConfigurationExport
```

Reply:
removed